app/controllers/idv/otp_verification_controller.rb

```diff
@@ -32,6 +32,7 @@ def update
 
       if result.success?
         idv_session.user_phone_confirmation = true
+        save_in_person_notification_phone
```

Comment:
Is there a link to the JIRA or mockups for this? I'd expect to see us checking a form field for permission to save the phone number like this

Reply:
The ticket is [LG-10123](https://cm-jira.usa.gov/browse/LG-10123). The [One Pager](https://docs.google.com/document/d/1PIhJ7ntS6-8odUS0EKJSwH0z5qOxxEIG-zdkfchBfjw/edit) does pose this question -- check out the Open Questions section

The plan is to put it behind a feature flag as we develop the feature and while it goes through reviews

Reply:
I think the question you're getting at is, "Should users opt-in to this new type of notification?"

This notification feature is intended to increase the security of the in-person flow so that it can be used app-wide, not just with specific partners. So making these notifications opt-in would decrease that security improvement

Reply:
Following up on the slack thread we had about this with @tomas-nava: I'll go ahead and merge this PR but also one of us will start a conversation with product to address your question/suggestion. We'll tag you so you can follow the conversation